content/guides/user-guides/multi-cloud/object-archival-restore/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archival and Retrieval
+description: "User guide for Object Archival and Retrieval"
+weight: 40
```

Comment:
Set the weight for each of the sections based on the order in which they must appear.

Reply:
Done